eng/scripts/get-aspire-cli.ps1

```diff
@@ -0,0 +1,489 @@
+#!/usr/bin/env pwsh
+
+[CmdletBinding()]
+param(
+    [string]$OutputPath = "",
+    [string]$Version = "9.0",
+    [string]$BuildQuality = "daily",
```

Comment:
I'm not sure I fully get the "Build Quality" thing. Can you explain what it is for?

Reply:
The idea is that you'd use that to select if you want latest from main branch, vs latest from a release branch for example. I'd probably switch the name to just `Quality` to mimic what is done in dotnet-install scripts: https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#options

![image](https://github.com/user-attachments/assets/b5f6ab27-5eda-4a54-89b5-2c852d3ab182)